src/node/rpc/frontend.h

```diff
@@ -579,24 +579,28 @@ namespace ccf
      * @param ctx Context for this RPC
      * @param input Serialised JSON RPC
      */
-    std::vector<uint8_t> process_pbft(
+    std::tuple<std::vector<uint8_t>, crypto::Sha256Hash> process_pbft(
       enclave::RPCContext& ctx, const std::vector<uint8_t>& input) override
     {
       // TODO(#PBFT): Refactor this with process_forwarded().
       Store::Tx tx;
+      crypto::Sha256Hash merkle_root;
 
       auto pack = detect_pack(input);
       if (!pack.has_value())
-        return jsonrpc::pack(
-          jsonrpc::error_response(
-            0,
-            jsonrpc::StandardErrorCodes::INVALID_REQUEST,
-            "Empty PBFT request."),
-          jsonrpc::Pack::Text);
+        return std::make_tuple(
```

Comment:
These tuples should be structs instead.

Reply:
This has been resolve. I am going to push this change as Olga is traveling and I am not sure when she will be able to look at this